mbuild/compound.py

```diff
@@ -43,15 +44,25 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
         Name of the file from which to load atom and bond information.
     relative_to_module : str, optional, default=None
         Instead of looking in the current working directory, look for the file
-        where this module is defined. This is typically used in Compound classes
-        that will be instantiated from a different directory (such as the
-        Compounds located in mbuild.lib).
+        where this module is defined. This is typically used in Compound
+        classes that will be instantiated from a different directory
+        (such as the Compounds located in mbuild.lib).
     compound : mb.Compound, optional, default=None
         Existing compound to load atom and bond information into.
     coords_only : bool, optional, default=False
         Only load the coordinates into an existing compoint.
-    rigid : bool, optional
+    rigid : bool, optional, default=False
         Treat the compound as a rigid body
+    **kwargs : keyword arguments
+        Arbitrary keyword arguments.
+
+    Args:
```

Comment:
However, load is it's own separate function, outside of `Compound`. Since `save` and its `**kwargs` are passed to other savers and such, would it be reasonable to have both defined?